src/joiDescribeTypes.ts

```diff
@@ -29,6 +29,10 @@ export interface BaseDescribe extends Joi.Description {
      * https://joi.dev/api/#objectunknownallow
      */
     unknown?: boolean;
+    /**
+     * https://joi.dev/api/?v=17.4.2#anycastto
+     */
+    cast?: string
```

Comment:
```suggestion
    cast?: 'string' | 'number'
```

Reply:
So I didn't want to do this initially because I was under the impression that any value could be passed to cast, but upon further review it can be `'string' | 'number' | 'map' | 'set'` and Joi enforces that each can only be called on certain types, so I've updated to reflect this.

I'm also working on a follow up PR to introduce support for map and set